packages/cli/test/cases/develop.default/import-map.snapshot.json

```diff
@@ -4157,7 +4156,7 @@
   "resolve-id-refs": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/resolve-id-refs/index.js",
   "sass/types": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/sass/types/index.d.ts",
   "sass/node": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/sass/sass.node.mjs",
-  "sass/default": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/sass/sass.default.js",
+  "sass": "/~/Users/owenbuckley/Workspace/project-evergreen/greenwood/node_modules/sass/sass.default.js",
```

Comment:
looks like this is corrected now
https://app.unpkg.com/sass@1.89.0/files/package.json